pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive `yacas` session:
+
+`yacas`
+
+- In the `yacas` session, execute a statement:
```

Comment:
```suggestion
-While in a `yacas` session, execute a statement:
```
Thoughts?

Remember to apply throughout.

Reply:
Done